src/util/assert.h

```diff
@@ -36,10 +36,10 @@ inline void mixxx_release_assert(const char* assertion, const char* file, int li
 #define ASSERT_FUNCTION __PRETTY_FUNCTION__
 #endif
 
-// If cond is false, produces a fatal assertion and quits ungracefully. Think
-// very hard before using this -- this should only be for the most dire of
-// situations where we know Mixxx cannot take any action without potentially
-// corrupting user data. Handle errors gracefully whenever possible.
+/** If cond is false, produces a fatal assertion and quits ungracefully. Think
+ * very hard before using this -- this should only be for the most dire of
+ * situations where we know Mixxx cannot take any action without potentially
+ * corrupting user data. Handle errors gracefully whenever possible. */
```

Comment:
use `///`

Reply:
With this style, it shows the documentation on mouseover for the Jetbrains IDE, idk how other IDEs do it. But I changed it now.

Reply:
Ah, this is also recognized by Clion, nvm :)